awc/src/lib.rs

```diff
@@ -4,24 +4,98 @@
     clippy::borrow_interior_mutable_const,
     clippy::needless_doctest_main
 )]
-//! An HTTP Client
+
+//! `awc` is a HTTP/WebSocket Client library implemented atop of `actix` stack.
```

Comment:
Don't want to suggest is is using the `actix` crate.

```suggestion
//! `awc` is a HTTP and WebSocket client library built using the Actix ecosystem.
```